lib/routemap_cli.c

```diff
@@ -155,6 +155,28 @@ void route_map_instance_show(struct vty *vty, struct lyd_node *dnode,
 		SKIP_RULE("ip next-hop prefix-len");
 		SKIP_RULE("source-protocol");
 		SKIP_RULE("source-instance");
+		/* OSPF specific match conditions. */
```

Comment:
If we have all route maps migrated, then this function is no longer necessary.

Reply:
Addressed